sdk/apis/tenancy/v1alpha1/types_workspacetype.go

```diff
@@ -116,6 +116,12 @@ type WorkspaceTypeSpec struct {
 	//
 	// +optional
 	DefaultAPIBindings []APIExportReference `json:"defaultAPIBindings,omitempty"`
+
+	// Mount is the mount configuration for the workspace type. Single type can have only one mount kind.
+	// If set, the workspace will be backed by a mount.
+	//
+	// +optional
+	Mount *metav1.TypeMeta `json:"mount,omitempty"`
```

Comment:
how is this related to workspace types?

Reply:
This would be something like `kind: mounts.contrib.kcp.io` so every workspace initiated via this type just needs to carry name only for the kind. basically bounts workspaceType to mount implementation kind. Still thinking if this is a good idea. 

But if we keep Kind of mount object inside workspace spec, workspace Type is not even used.

maybe at the workspace level:
`Type WorkspaceTypeReference` should become with `Kind: Mount,LogicalCluster` ?

Reply:
something like:
```
spec:
  URL: https://192.168.3.55:6443/clusters/root:consumer
  cluster: ofebwh0c93o6swd3
  type:
    name: organization
    path: root
```

Would become:
```
spec:
  URL: https://192.168.3.55:6443/clusters/root:consumer
  cluster: ofebwh0c93o6swd3
  type:
    kind: WorkspaceType
    name: organization
    path: root
```

and 
```
spec:
  URL: https://192.168.3.55:6443/clusters/root:consumer
  cluster: ofebwh0c93o6swd3
  type:
    kind: Mount
    name: mountBob
    kind: mounts.contrib.kcp.io
    version: v1alpha1
```

Reply:
In my mind type and mount are mutual exclusive, i.e. WorkspaceTypes are actually LogicalClusterTypes.

Reply:
So do you think its root level new field in Workspace? Any suggestions how this could be named considering type is taken? :) Or just `mount:`